### PR TITLE
Remove installnamespace value from linkerd-controle-plane chart

### PR DESCRIPTION
**Pull request by mikutas**
I guess it was missed to remove.

### CODE REVIEW

**Review by alpeb — APPROVED** [review]
Great find, thanks :-)

File: charts/linkerd-control-plane/values.yaml [hunk 1]
```diff
@@ -379,10 +379,6 @@ policyValidator:
 # - The external tool needs to create the namespace with the label:
 #     config.linkerd.io/admission-webhooks: disabled
```
Comment (kleimkuhler) [inline charts/linkerd-control-plane/values.yaml:380]: It looks like this comment section being above `nodeSelector` is now messing with how it gets rendered in `README.md`. We don't want those changes as part of this.

@alpeb we're okay to remove this comment section as well though I think?

Comment (alpeb) [reply charts/linkerd-control-plane/values.yaml:380]: Ah indeed, that needs to go

**Comment by mikutas** [comment]
OK I restored it but placed it below `nodeSelector` section because if I place commented out `spValidator` section  above nodeSelector, `bin/helm-docs` makes README like this...

```markdown
| nodeSelector."kubernetes.io/os" | string | `"linux"` |  |
```

**Review by kleimkuhler — APPROVED** [review]
Thanks @mikutas!

**Pull request merged**